documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+## Restricted Items
+1. Network ranges and IP addresses <br>
+The organisation currently have these classed as _sensitive_
+1. S3 Bucket names <br>
+Bucket names for a public facing endpoint. Bucket names are typically a hash to keep them obscured, putting them in a document would give context and remove the ambiguity.
+2. Individuals names, contact details or any other personally identifiable information.
```

Comment:
`2. Individuals` -> `1. Individual's` - GitHub specifies numbered list entries as starting with 1. regardless of their position in the list.  This makes reordering list entries really easy, as you don't have to manually renumber.